docker-compose.yml

```diff
@@ -364,15 +364,15 @@ services:
     ports:
       - "8084:80"
     environment:
-      - ADMIN_EMAIL=john_doe@there.com
-      - ADMIN_PASS=sp1
-      - IDP_NAME=THIS VARIABLE IS REQUIRED BUT PROBABLY NOT USED
-      - SECRET_SALT=NOT-a-secret-k49fjfkw73hjf9t87wjiw
-      - SECURE_COOKIE=false
-      - SHOW_SAML_ERRORS=true
-      - SAML20_IDP_ENABLE=false
-      - ADMIN_PROTECT_INDEX_PAGE=false
-      - THEME_USE=default
+      ADMIN_EMAIL: john_doe@there.com
+      ADMIN_PASS: sp1
+      IDP_NAME: THIS VARIABLE IS REQUIRED BUT PROBABLY NOT USED
+      SECRET_SALT: NOT-a-secret-k49fjfkw73hjf9t87wjiw
+      SECURE_COOKIE: "false"
+      SHOW_SAML_ERRORS: "true"
+      SAML20_IDP_ENABLE: "false"
+      ADMIN_PROTECT_INDEX_PAGE: "false"
+      THEME_USE: material:material
```

Comment:
Just checking that this change was intentional.

Reply:
Not really. It should have been in the next PR. But since tests pass, it's ok to leave it like this.